pymc3/tests/test_distributions.py

```diff
@@ -1030,14 +1030,19 @@ def test_half_normal(self):
             lambda value, sigma: sp.halfnorm.logcdf(value, scale=sigma),
         )
 
-    @pytest.mark.xfail(reason="Distribution not refactored yet")
-    def test_chi_squared(self):
+    def test_chisquared(self):
         self.check_logp(
             ChiSquared,
             Rplus,
-            {"nu": Rplusdunif},
+            {"nu": Rplus},
```

Comment:
I see now, I think the previous domain was more appropriate as it was positive + discrete.